client/blocks/comment-detail/index.jsx

```diff
@@ -201,6 +203,7 @@ export class CommentDetail extends Component {
 			postAuthorDisplayName,
 			postId,
 			postTitle,
+			preventServerRequest,
```

Comment:
Optional, but how about `refreshCommentData`? So we're clear on what requests we're talking about.